ly/words.py

```diff
@@ -167,12 +171,14 @@
     'dashDot',
     'dashHat',
     'dashLarger',
+    'dashBang', # since 2.20
```

Comment:
Nitpick: can you move it up so the alphabetical order is maintained?

Reply:
Oops.  Yes.  Done.